### ISSUE TITLE
Replace goodreads links with storygraph ones

### ISSUE THREAD

**Issue by sentientist-tech** [aborazmeh/awesome-palestine#37]
Thanks for this awesome project!

I was thinking if we could replace the goodreads links of the books listed with storygraph.

Reason: Goodreads is owned by Amazon, [who is a huge investor in Israel](https://boycott.thewitness.news/target/amazon)

**Comment by aborazmeh** [comment]
My reason at first to prioritize Goodreads was that it has a large database. I started this project with localization in mind, and Goodreads is the largest database for books and editions online. The database is built by thousands of librarian volunteers from all over the world, and being a librarian myself it makes it flexible to add/edit missing editions on need.
I tried The StoryGraph some years ago, and it seems to take advantage of Goodreads shortages, like the bad recommendations, and half-stars, but I am not sure about its database and how large it is, and how to add missing editions. (Goodreads book reviews are still much better, though)

Anyway, your reasoning to use alternatives for this project is valid and fair. Maybe we can add links to The StoryGraph or other services (like the official websites for books which have them) for some books, and only use Goodreads when there are no alternatives.

Thank you for your contributions

**Comment by Zain-ul-din** [comment]
@aborazmeh can we provide a link of the `zlib`. i've a `tor` link

**Comment by aborazmeh** [comment]
> @aborazmeh can we provide a link of the `zlib`. i've a `tor` link

No, we can't. It is illegal, and whoever wants to find these enlisted resources will be able to (physically or online)
